functions-app/.nvmrc

```diff
@@ -1 +1 @@
-v10.14.1
+v10.16.3
```

Comment:
is there a particular reason for this?  i ask because 14.1 is recommended by MS

Reply:
Overzealous version upgrading 🙄 Now reverted.